scripts/protocolOwnershipVerifier.ts

```diff
@@ -81,9 +81,10 @@ export const verifyOwnership = async () => {
 async function verifyArbOwner(provider: Provider, l2Executor: string) {
   const ownerPrecompile = ArbOwner__factory.connect(ARB_OWNER_PRECOMPILE, provider);
 
+  assert(await ownerPrecompile.isChainOwner(l2Executor), "L2Executor should be the chain owner");
+
   const owners: string[] = await ownerPrecompile.getAllChainOwners();
```

Comment:
I dont know if this will fail now or not? Maybe lets just remove it since we know tihis func is broken

Reply:
I think it shouldn't fail, previously it failed only at `owners[0] == l2Executor`,  length check was successful

Reply:
```suggestion
  const owners: string[] = await ownerPrecompile.getAllChainOwners();
```